onnxruntime/contrib_ops/webgpu/quantization/dp4a_matmul_quantize.wgsl.template

```diff
@@ -0,0 +1,84 @@
+//  Template for DP4A Matrix Multiply Quantization
+// Quantizes input matrix A for DP4A computation
```

Comment:
nit: extra space

We should add the copyright header

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

Reply:
Done